scripts/nagios_checks/autoreduce_checkconsumers.py

```diff
@@ -36,6 +36,4 @@ def checkConsumer():
     return 0
 
 
-# pylint: disable=using-constant-test
```

Comment:
I think nagios runs `checkConsumer` directly for example. That `if "__name__":...` bit of code was probably put in for manual user testing. I would replace it with `if __name__ == "__main__":`

Reply:
This was my thought to, but I wasnt sure how it would effect nagios so I wanted to check